cmd/crictl/container.go

```diff
@@ -385,6 +392,21 @@ var removeContainerCommand = &cli.Command{
 				logrus.Errorf("removing container %q failed: %v", id, err)
 				errored = true
 				continue
+			} else if !ctx.Bool("keep-logs") {
+				logPath := resp.GetStatus().GetLogPath()
+				if logPath != "" {
+					logRotations, err := filepath.Glob(logPath + "*")
```

Comment:
Is this save for the log rotation? We would also possible remove files which are not rotated logs but other files, right?

Reply:
It's possible, although the rotations attach to the file name so it's unlikely, e.g. this was from my test with cri-o:
```
-rw------- 1 root root 5.3M Feb 26 05:49 0.log
-rw-r--r-- 1 root root 1.2M Feb 26 05:44 0.log.20240226-053352.gz
-rw------- 1 root root  11M Feb 26 05:44 0.log.20240226-054404
```
Another container would be e.g. 1.log, but a user could have backed up a 0.log.bak, and this code would delete that too. I wasn't sure if the log rotation always follows this format though.
Reading https://kubernetes.io/docs/concepts/cluster-administration/logging/ the rotation in /var/log should only be performed by kubelet, so I could use that as a guide, removing files in this format in that directory and ignoring rotations outside it.

Reply:
It looks like kubelet uses a very similar pattern: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/logs/container_log_manager.go#L312
For the time being I'll do the same and just add the missing dot; I'm averse to adding hardcoded references to the dir, especially because it seems a bit more variable under Windows (the doc mentions C:\var\logs and C:\var\log\pods).